api.py

```diff
@@ -52,10 +52,10 @@ def get_geocoords(address):
 		# and its corresponding geocoords in its own table in the event that, say, 
 		# there are many different users living in the same apartment building. 
 		# This would avoid the repetition of querying geocoords for the same street address each time.
-		return (37.6,-95.665)
+		return 37.6, -95.665
 
 
-def call_api(**kwargs):
+def google_geocoding(address=None, city=None, state=None):
```

Comment:
Naming is hard. But a function with a good name help understand what is does. 
Default value params are really useful when some params are not required. that way I can easily use this method and remove all the kwargs logic bellow.